src/vs/workbench/common/editor/untitledTextEditorModel.ts

```diff
@@ -205,4 +205,8 @@ export class UntitledTextEditorModel extends BaseTextEditorModel implements IUnt
 	isReadonly(): boolean {
 		return false;
 	}
+
+	unregisterWorkingCopy() {
```

Comment:
every change we make in the `vs` folder should be prefixed with a comment `// {{SQL CARBON EDIT}}` so we can easily identify our customizations to the upstream codebase.  Please make this update to changes under `vs`.